From cad8ae2986ab0b70d0f13b96c98f340564213d4c Mon Sep 17 00:00:00 2001
From: Li Jun <B47624@freescale.com>
Date: Mon, 29 Sep 2014 10:04:45 +0800
Subject: [PATCH] MLK-9617-7 usb: otg: add hnp_polling flag in struct otg_fsm

This patch adds a flag hnp_polling to indicate if HNP polling is supported.

Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit b19d4b1110fe1fb2ec3141233781a1e016c9a9cf)
---
 include/linux/usb/otg-fsm.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/usb/otg-fsm.h b/include/linux/usb/otg-fsm.h
index e329de6..1b0f601 100644
--- a/include/linux/usb/otg-fsm.h
+++ b/include/linux/usb/otg-fsm.h
@@ -109,6 +109,7 @@ struct otg_fsm {
 	int b_srp_done;
 	int b_hnp_enable;
 	int a_clr_err;
+	int hnp_polling;
 
 	/* OTG test device */
 	int tst_maint;
-- 
1.8.0

